src/libraries/System.Formats.Tar/src/System/Formats/Tar/PaxTarEntry.cs

```diff
@@ -100,6 +103,9 @@ public PaxTarEntry(TarEntryType entryType, string entryName, IEnumerable<KeyValu
         /// <summary>
         /// Initializes a new <see cref="PaxTarEntry"/> instance by converting the specified <paramref name="other"/> entry into the PAX format.
         /// </summary>
+        /// <exception cref="ArgumentException"><para>Cannot convert a PAX Global Extended Attributes entry.</para>
```

Comment:
From https://github.com/dotnet/runtime/pull/73394#discussion_r941847904
> The rule for the sentence fragments in exceptions is to word them as if they were preceded them with "Throws {type} when ".

So I think this should be 
```suggestion
        /// <exception cref="ArgumentException"><para><paramref name="other"/> is a PAX Global Extended Attributes entry and cannot be converted.</para>
```

Reply:
Sure. The other conversion APIs have similar wording. Do you want to copy the text, or do you want me to do it?

Reply:
I'll do it.